services/utils/__init__.py

```diff
@@ -6,12 +6,15 @@
 from multidict import MultiDict
 from urllib.parse import urlencode, quote
 from aiohttp import web
+from enum import Enum
 from functools import wraps
 from typing import Dict
 import logging
 import psycopg2
 from packaging.version import Version, parse
 
+USE_SEPARATE_READER_POOL = os.environ.get("USE_SEPARATE_READER_POOL", "0")
```

Comment:
could be cast to a boolean instead to make using it a bit cleaner.

Reply:
I'll also echo the comment that @dhpollack made on the need for a separate toggle for the reader pool, vs. simply looking at the existence of the `MF_METADATA_DB_READ_REPLICA_HOST`? This would cut down on the environment variables required to configure the feature as well.